src/components/fields/SchemaField.js

```diff
@@ -185,6 +187,8 @@ function SchemaFieldRender(props) {
     case 'array':
       displayLabel =
         isMultiSelect(schema, definitions) || isFilesArray(schema, uiSchema, definitions);
+      // If it's an array of objects, use a div element
+      useDiv = !!schema.items && schema.items.type === 'object';
```

Comment:
Is it more readable to wrap that expression in `( ... )` ?

Reply:
Yes, probably if I do:

```js
useDiv = !!schema.items && (schema.items.type === 'object');
```